### ISSUE TITLE
G doesn't go to bottom of long list of items

### ISSUE THREAD

**Issue by folofjc** [Wojnach/vimflowy#20]
I have a page with a very long list of items that is longer than one page. `G` does not go to the bottom. It doesn't go to the bottom of the page, either. `GG` does not either. However, if I do `G`, then wait till it jumps, then `G` again, then it will go to the bottom.

**Comment by Wojnach** [comment]
Yeah this is a limitation I ran into back when It was written. I haven't figured out a workaround for it yet. I tried a bunch of different approaches but I ended up with new problems down the line when I did that. 

The main problem is that the items off screen aren't _always_ focusable. You can move to them but most of the time you lose cursor focus. The current approch is to move to the bottommost focusable item and then collapse the rest of the items to make new items appear. So If you want to get to the bottom of a long list you end up spamming GG. 

I'll have another look to see if I can figure something out. Maybe we can fix it with some Zoom-in/out black magic.

**Comment by folofjc** [comment]
Okay, I figured there was something like this. This one is definitely low priority, since spamming GG is not that big of a deal.

**Comment by akak0000** [comment]
@Wojnach Also would love a fix for this if you think of anything. Thanks!

**Comment by akak0000** [comment]
@Wojnach Maybe you could use a recursive setTimeout so when "G" is pressed, it goes to the current bottom, and if you detect that there are more unrendered elements, schedule a setTimeout in 0 seconds to do the same thing again, in order to give the browser time to render and return control to the javascript and deliver another "G". Does that make sense?
